components/kyma-environment-broker/internal/error/errors.go

```diff
@@ -170,3 +170,27 @@ func checkK8SError(cause error) LastError {
 
 	return lastErr
 }
+
+// UnwrapOnce accesses the direct cause of the error if any, otherwise
+// returns nil.
+func UnwrapOnce(err error) (cause error) {
```

Comment:
I know @PK85 said he is not interested in maintaining backward compatibility, but I want to advocate one last time we make sure not to break these two functions as there are metrics and alerts depending on them that may break
https://github.com/kyma-project/control-plane/blob/d5b5786ba3de3d53a74a80d291b7492060c2322d/components/kyma-environment-broker/internal/error/errors.go#L145
https://github.com/kyma-project/control-plane/blob/d5b5786ba3de3d53a74a80d291b7492060c2322d/components/kyma-environment-broker/internal/error/errors.go#L90

Reply:
should be fine, but will check once again